### PR TITLE
Add option to delete files after export #247

### PR DESCRIPTION
**Pull request by BIBLETUM**
**Description:**

*Add option to delete files after export*
#247 

**Tasks:**
- [ ] Add option to delete files after export to settings layout
- [ ] Add option to delete files after export on ImageViewFragment
- [ ]  Add option to delete files after export on GalleryFragment
- [ ] Translate all new strings

### CODE REVIEW

File: app/src/main/java/dev/leonlatsch/photok/imageviewer/ui/ImageViewerViewModel.kt [hunk 1]
```diff
@@ -98,7 +98,10 @@ class ImageViewerViewModel @Inject constructor(
             currentPhoto ?: return@launch
             currentPhoto!!.id ?: return@launch
 
-            val success = photoRepository.exportPhoto(currentPhoto!!)
-            if (success) onSuccess() else onError()
+            photoRepository.exportPhoto(currentPhoto!!).let {
```
Comment (leonlatsch) [inline app/src/main/java/dev/leonlatsch/photok/imageviewer/ui/ImageViewerViewModel.kt:101]: name "it" as "success"

File: app/src/main/res/values-de/strings.xml [hunk 2]
```diff
@@ -100,11 +100,13 @@
     <!-- EXPORT -->
     <string name="export_are_you_sure">Bist du sucher, dass du %1$d Dateien, zurück in deine Galerie, <b>exportieren</b> willst?</string>
     <string name="export_are_you_sure_this">Bist du sicher, dass du diese Datei, zurück in deine Galerie, <b>exportieren</b> willst?</string>
+    <string name="export_and_delete_are_you_sure_this">Sind Sie sicher, dass Sie diese Datei in Ihre Galerie <b>exportieren</b> und löschen möchten?</string>
```
Comment (leonlatsch) [inline app/src/main/res/values-de/strings.xml:103]: In german we use "du" and here are some grammar issues

Comment (leonlatsch) [reply app/src/main/res/values-de/strings.xml:103]: Also, please don't use google translate for these translations. Add the english with a  in order for the badges in the README to work.

**Review by leonlatsch — CHANGES_REQUESTED** [review]

**Review by leonlatsch — CHANGES_REQUESTED** [review]
Very nice, BUT the message for exporting multiple images still needs to be adjusted. Its enough to do it for the new gallery and not the old one

**Comment by leonlatsch** [comment]
@BIBLETUM and maybe you can find other icons for these options. Hard to imagine but maybe you find something in the assets of android studio. If not its ok to leave ic_delete for both

**Comment by BIBLETUM** [comment]
Unfortunately could not find any other matching icons

**Review by leonlatsch — APPROVED** [review]
👍

File: app/src/main/res/values-es/strings.xml [hunk 3]
```diff
@@ -249,5 +254,5 @@
     <!-- Recovery Menu -->
     <string name="recovery_subtitle">Recovery Menu</string> <!-- TODO -->
     <string name="recovery_menu_item_open_photok">Open Photok</string> <!-- TODO -->
-    <string name="recovery_subtitle_reset_hide_app">Reset Hide App Setting</string> <!-- TODO -->
```
Comment (leonlatsch) [inline app/src/main/res/values-es/strings.xml:252]: this TODO needs to stay

**Pull request merged**